### PR TITLE
Add additional support of 'yaml' file extension

### PR DESCRIPTION
**Pull request by chclaus**
Hej hej,

following the official naming convention of [yaml.org](http://yaml.org/faq.html), I suggest that the extension 'yaml' should also be supported.

Best,
Christian

### CODE REVIEW

File: testfixtures.go [hunk 1]
```diff
@@ -268,7 +268,7 @@ func fixturesFromFolder(folderName string) ([]*fixtureFile, error) {
 	}
 
 	for _, fileinfo := range fileinfos {
-		if !fileinfo.IsDir() && filepath.Ext(fileinfo.Name()) == ".yml" {
+		if !fileinfo.IsDir() && (filepath.Ext(fileinfo.Name()) == ".yml" || filepath.Ext(fileinfo.Name()) == ".yaml") {
```
Comment (andreynering) [inline testfixtures.go]: What about caching the result of `filepath.Ext(fileinfo.Name())` into its own variable?

**Review by andreynering — APPROVED** [review]
Hey @chclaus, thanks for opening this PR.

A small code nit and this is good to go.

File: testfixtures.go (hunk 1 above)
Comment (chclaus) [reply testfixtures.go]: Hej,

you're absolutely right! 🙈

**Comment by andreynering** [comment]
Thanks!

Released in v2.5.2

**Pull request merged**